scripts/gh-predeploy.js

```diff
@@ -456,8 +456,7 @@ function prepareExample(exampleDir, isLocal, done) {
   // Cancel early if the package.json says this is not a demo.
   const {blocklyDemoConfig: demoConfig} = packageJson;
   if (!demoConfig) {
-    done();
-    return;
+    return done();
```

Comment:
I don't think this is correct; we don't want to return the value of this function (I don't even know what that value would be, it's just arcane gulp magic).

If we're returning early then we actually shouldn't return a gulp task. The types here are super unimportant. I think you can change the return type to be `{*}` or maybe `{Function | undefined}`